plugins/power/tasmota_

```diff
@@ -0,0 +1,155 @@
+#!/bin/sh
+#
+# Copyright (C) 2019-2022 Tim Connors
+# GNU Library General Public License v2 or later
+#
+#
+# munin plugin that logs voltage, energy usage, power, power factor and current logged from tasmota power switches
+#
+# To install the plugin, copy or move the plugin to /usr/share/munin/plugins/ set
+# the chmod to 755 and create a symbolic link:
+# ln -s /usr/share/munin/plugins/tasmota_ /etc/munin/plugins/tasmota_hostname_voltage
+# ln -s /usr/share/munin/plugins/tasmota_ /etc/munin/plugins/tasmota_hostname_energy
+# ln -s /usr/share/munin/plugins/tasmota_ /etc/munin/plugins/tasmota_hostname_power
+# ln -s /usr/share/munin/plugins/tasmota_ /etc/munin/plugins/tasmota_hostname_powerfactor
+# ln -s /usr/share/munin/plugins/tasmota_ /etc/munin/plugins/tasmota_hostname_current
+#
+# Plugin also implements suggests, so if you have /etc/genders
+# populated with "tasmota,powermon" flags for those tasmota devices
+# that implement energy monitoring, and nodeattr installed, you can
+# run an ansible play like this to set up your links:
+# https://github.com/spacelama/ansible-initial-server-setup/tree/master/roles/monitoring/tasks
+#
+#%# family=auto
+#%# capabilities=autoconf
+
+SWITCH=$(basename "$0" | cut -d_ -f2)
+FUNCTION=$(basename "$0" | cut -d_ -f3)
+
+if [ "$1" = "autoconf" ]; then
+    echo yes && exit 0
+fi
+
+if [ "$1" = "suggest" ]; then
+    nodeattr -n 'tasmota&&powermon' | while read device ; do
```

Comment:
Please use `read -r` here. See the output of `shellcheck -s dash FILENAME` for more hints.

Reply:
Done